src/app_websocket_inner.rs

```diff
@@ -43,8 +40,7 @@ impl AppWebsocketInner {
                 while let Ok(msg) = rx.recv::<AppResponse>().await {
                     if let holochain_websocket::ReceiveMessage::Signal(signal_bytes) = msg {
                         let mut event_emitter = mutex.lock().await;
-                        let signal = Signal::try_from_vec(signal_bytes).expect("Malformed signal");
-                        event_emitter.emit("signal", signal);
+                        event_emitter.emit("signal", signal_bytes);
```

Comment:
This actually uses `bincode` to serialize its input and the Signal type is no longer supported after the format changes we made recently. Subtle test break here where we couldn't send signals any more